packages/client/src/environments/environment.prod.ts

```diff
@@ -2,8 +2,20 @@ export const environment = {
   production: true,
   serverUrl: 'localhost:3000',
   movement: {
-    runningSpeed: 0.3,
-    walkingSpeed: 0.15,
+    runningSpeed: 0.4,
```

Comment:
ERROR in /home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-map/keyboard-control/keyboard-control.component.ts (60,40): Property 'crawlingSpeed' does not exist on type '{ runningSpeed: number; walkingSpeed: number; }'.

  ERROR in /home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-map/world/world.component.ts (25,33): Property 'tiles' does not exist on type '{ production: boolean; serverUrl: string; movement: { runningSpeed: number; walkingSpeed: number;...'.

  ERROR in /home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-map/world/world.component.ts (26,34): Property 'load3dTiles' does not exist on type '{ production: boolean; serverUrl: string; movement: { runningSpeed: number; walkingSpeed: number;                                           ...'.

  ERROR in /home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-map/world/world.component.ts (70,21): Property 'loadTerrain' does not exist on type '{ production: boolean; serverUrl: string; movement: { runningSpeed: number; walkingSpeed: number;                                           ...'.

  ERROR in /home/ubuntu/CesiumFPS/packages/client/src/app/game/views/game-map/world/world.component.ts (82,19): Property 'terrain' does not exist on type '{ production: boolean; serverUrl: string; movement: { runningSpeed: number; walkingSpeed: number;...'                                           .